### DISCUSSION TITLE
Logos Round 2! [Deadline: Feb 2]

### DISCUSSION THREAD

**Discussion by ebullient** [commonhaus/foundation#57] [Consensus Building]
Focus on shape and theme/concept, not color. We're trying to narrow it down from 4 concepts to 2 (even better would be 1).

<img src="https://github.com/commonhaus/foundation-draft/assets/808713/f1749541-b03a-41bd-b197-a10582703c68" width="100%" />

[Additional positioning and sizes](https://docs.google.com/presentation/d/1E4DqWuQGkborPXXWfwGJgVgf2N--V94HacFsptEbKFA) 

We'll narrow the poll question: 

1. Choose your favorite concept (row: A, B, C, D).
2. (🚀, optional) Specific number if you have a favorite
3. (👎, optional) Specific number if it is an absolute no

Constructive comments help. 

We can further explore a concept once we narrow to a favorite (and if we haven't otherwise picked a winner).

**Comment by insectengine** [comment]
Zoomed in with details.

## Category A (Row 1)

![commonhaus_r2_large_1](https://github.com/commonhaus/foundation-draft/assets/928656/567086ee-0f0c-45ba-9adc-10562f8c9977 "A 1")
![commonhaus_r2_large_2](https://github.com/commonhaus/foundation-draft/assets/928656/b09ee55a-6e03-4483-a924-87e3b3636ebb "A 2")
![commonhaus_r2_large_3](https://github.com/commonhaus/foundation-draft/assets/928656/df1abcdf-32d1-4fa7-b18d-d8c73aecfccc "A 3")
![commonhaus_r2_large_4](https://github.com/commonhaus/foundation-draft/assets/928656/852b805f-2461-4bed-883d-9b8241fa5d68 "A 4")

## Category B (Row 2)

![commonhaus_r2_large_5](https://github.com/commonhaus/foundation-draft/assets/928656/eb772049-d1a3-477a-83b3-c32e317f7255 "B 5")
![commonhaus_r2_large_6](https://github.com/commonhaus/foundation-draft/assets/928656/37b02af3-b7cc-4032-8ebb-b3b1020055a2 "B 6")
![commonhaus_r2_large_7](https://github.com/commonhaus/foundation-draft/assets/928656/83ff71d7-5bbd-46de-b98a-450d306912f0 "B 7")
![commonhaus_r2_large_8](https://github.com/commonhaus/foundation-draft/assets/928656/1aab0fa0-9c29-4d11-89ab-36478438ad60 "B 8")

## Category C (Row 3)

![commonhaus_r2_large_9](https://github.com/commonhaus/foundation-draft/assets/928656/c1153252-160f-46d5-8373-79d09091844f "C 9")
![commonhaus_r2_large_10](https://github.com/commonhaus/foundation-draft/assets/928656/1635a952-1a3b-4235-b3fb-3ff788cd5d43 "C 10")
![commonhaus_r2_large_11](https://github.com/commonhaus/foundation-draft/assets/928656/c983e3d6-583c-4b1a-8090-a2c97932a115 "C 11")
![commonhaus_r2_large_12](https://github.com/commonhaus/foundation-draft/assets/928656/5695d3df-e289-4a73-bc3b-54d26a32acaa "C 12")

## Category D (Row 4)

![commonhaus_r2_large_13](https://github.com/commonhaus/foundation-draft/assets/928656/92725a4d-ba1e-4873-8952-d05d66bacc8b "D 13")
![commonhaus_r2_large_14](https://github.com/commonhaus/foundation-draft/assets/928656/c63b7336-cb71-4c9b-a2eb-d4048ba68ecd "D 14")
![commonhaus_r2_large_15](https://github.com/commonhaus/foundation-draft/assets/928656/0c2c7993-2cc6-4ce8-b3ff-488e6a9c06d9 "D 15")
![commonhaus_r2_large_16](https://github.com/commonhaus/foundation-draft/assets/928656/6c150b43-2dbe-45d3-93d3-98c7228bcb83 "D 16")
![commonhaus_r2_large_17](https://github.com/commonhaus/foundation-draft/assets/928656/b530d88d-cf80-43f6-991a-00148e2973ce "D 17")

**Comment by maxandersen** [comment]
5,6 and 8. 

Fyi: cant Access the doc without red hat login

So now James comment shows up: B. 

Dont like Any of the others as i prefer organic :)

**Comment by ebullient** [comment]
We're already in trouble from a narrow down point of view. :-P

1. concept: **A** (I like stained glass)
2. 🚀:  2. It has a "nestled in" feeling that I dig.
3. 👎: 17. It has the totally wrong feel for me.

**Comment by kenfinnigan** [comment]
1. concept A (though B is a close second as it's reminicent of [LiveOak's logo](https://github.com/liveoak-io), and I'm only thinking of this now, but maybe there's an opportunity for a variation of B where the roots are actually in the ground like LiveOak to have the roots not be in nothing)
2. 🚀: 1 or 2. (Note, I prefer 1 over 3 because of the rounded doorway)

**Comment by cowtowncoder** [comment]
1. Concepts A and B preferred (not sure which one I prefer of the two)
2. A2 or B8

**Comment by sigrunixia** [comment]
Just like a flag, a logo should be something a child can draw. 

1. Row B, Row D should also be there and present on official documents. 
1.a. As much as I like A, in smaller sizes, those extra stained glass lines will get lost. :(
2. 🚀 5, 6, 8
2.a. Of Row A, 2 is my favorite. If they can simplify the stained glass and maybe build the house more into it, I would jump for it in a heartbeat.
2.b. For Row D "more official" icon, 13 and 16 look the best to my eye- rule of thirds maybe. 
3. 👎 7 and 17.

**Comment by angellk** [comment]
🚀 Concept A
🚀 Concept B
👎 Concept C
👎 Concept D

🚀 A2 - i love the concept of this one and how the path leads to the house and the door connects to the path as well
🚀 B6 - love the simplicity - would look great anywhere

**Comment by Sanne** [comment]
A series is my favourite; I particularly love A1, would look great in glass, perhaps some colour.
   Minor drawback of A series: it might not be very clear that it's a house in a tree.

All Bs are nicely playful sa the treehouse concept is more explicit, except I'd avoid B8 as it's overly complex; they are less fascinating that As though.

C series is acceptable, however I don't like C12 as the "house" is not visible which is the only point in favour compares to A series, which I'd highly prefer.

D series doesn't speak to me.

HTH

**Comment by jonatan-ivanov** [comment]
I think a logo should be simple, I like B and A (B > A) but not so much C and D.
In the spirit of being simple: B6, B5, A4, A2.
(I know that this is not about colors but I would be careful choosing them in a way so that branches in A do not look like roads and trunk in B does not look like an explosion catapulting the house into the clouds.)

**Comment by gavinking** [comment]
I conferred with my wife, and our position is -1 to the broccoli and to the churches.

As a first impression, I like A1, and A4.

**Reply by ebullient** [reply]
broccoli and churches.. 🤣🤗

**Comment by cealsair** [comment]
- My favorite concept is D
- My favorite design is D-17

**Comment by aalmiray** [comment]
🚀 D17
Runner ups: A1, B8

**Comment by aalmiray** [comment]
I just discovered https://github.com/tree-sitter/tree-sitter and its logo 
![tree-sitter-small](https://github.com/commonhaus/foundation-draft/assets/13969/44a092f3-c800-4fe8-bf44-4769a3af5b03)

**Reply by ebullient** [reply]
I am telling you. Everyone loves treehouses. ;)

**Comment by ebullient** [comment]
Closed for now. Next round soooon. ;)
